agent/containers/images/Makefile

```diff
@@ -259,7 +326,15 @@ $(foreach tagtype,${_TAG_TYPES},$(eval $(call _TAG_RULE,${tagtype})))  # Define
 
 # Build the tags file for the given distribution. (We rename "centos" to "epel".)
 %-tags.lis:
+ifdef CI
+	if [[ -z "${_RPM_PATH}" ]] ; then echo "No RPM found with ID ${_LOCAL_GIT_HASH} for ${_LOCAL_MAJORMINOR} in ${CI_RPM_LOC}" >&2 ; exit 2 ; fi
+	if grep ' ' <<< ${_RPM_PATH} ; then echo "Multiple RPMs found:  ${_RPM_PATH}" >&2 ; exit 2 ; fi
+	printf -- "%s\n" ${_LOCAL_GIT_HASH} > ${@}
+	printf -- "v%s\n" "$(shell grep -Eo '[0-9]+\.[0-9]+\.([0-9]+(-[0-9]+)?)?' <<< "${_RPM_PATH}")" >> ${@}
+	printf -- "v%s-latest v%s-latest\n" ${_LOCAL_MAJOR_VER} ${_LOCAL_MAJORMINOR} >> ${@}
+else
```

Comment:
Don't we need to do the `./gen-tags-from-rpm` in the CI case as well?

Reply:
`gen-tags-from-rpm` is a fairly simple shell script which culminates in the same three `printf`'s as you cited here.  In the case of the script, the values filled in come from a `yum` command which targets a COPR repository.  In the CI case, we built the RPM locally, so we fill in the values from the local context, instead.

A number of the helper scripts like `gen-tags-from-rpm` could probably be in-lined in the `Makefile`.  I've been trying to resist that temptation.  I've also been trying to resist the temptation to screw with the scripts themselves.  So, the result is the `if`/`else` construct that troubled you, here, where I've split the difference.

Reply:
Don't we want to move away from COPR as the place we derive RPMs from for containers?  I had thought we want to build the RPMs for targets in build containers, and then use those RPMs to construct the container images we want.

Reply:
Yes:  the CI builds the RPMs locally, and then it (will) use(s) those RPMs to build containers; the legacy release process uses COPR to build the RPMs, and then it uses _those_ RPMs to build containers.  So, this PR is expanding the capability of this makefile to be able to support the CI model as well as the legacy model.